### PR TITLE
Blog Post about Random Slack Avatars

### PR DESCRIPTION
**Pull request by nerddtvg**
Building on an idea from Phil, this shows how to use Azure Functions and the Slack API to change a user's Slack photo with a random photo from https://thispersondoesnotexist.com/

### CODE REVIEW

**Comment by nerddtvg** [comment]
I think I'm finally done making modifications now.

File: _posts/2021-07-13-randomize-slack.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+After your app is created, it will appear in your apps list at [https://api.slack.com/apps](https://api.slack.com/apps). Open that app and browse to "OAuth & Permissions" in the menu. Copy the `User OAuth Token` presented at the top of the page. It must begin with `xoxp-` to be valid.
+
+### Azure Infrastructure
+
+Our [Azure Function App](https://docs.microsoft.com/en-us/azure/azure-functions/) only needs the function app itself, a storage account, and an app service plan. We will utilize the consumption plan to keep costs [extremely low](https://azure.microsoft.com/en-us/pricing/details/functions/).
+
+#### Setup the Azure Infrastructure
+
+You can utilize the portal to configure you're own Azure Function App, however we have provided a template you can use. You will need to change the names of the resources however.
```
Comment (Techneaux) [inline _posts/2021-07-13-randomize-slack.md:69]: typo

Comment (Techneaux) [reply _posts/2021-07-13-randomize-slack.md:69]: `you're` to your

**Review by Techneaux — CHANGES_REQUESTED** [review]
Found one typo. Love the tutorial.

File: _posts/2021-07-13-randomize-slack.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+subtitle:    ""
+date:        2020-07-13 12:00
+author:      "Drew Garrett"
+headerImg:   "/assets/images/posts/code.jpg"
+description: ""
+---
+
+## Randomize Your Slack Avatar with Azure Functions
+
+I enjoy having some fun with my Slack avatars which are almost never a photo of myself. With inspiration from [Phil Cryer's post]({% post_url 2021-07-13-randomize-slack %}) last year, I decided to implement a random avatars using an [Azure Function](https://docs.microsoft.com/en-us/azure/azure-functions/) and images from [https://thispersondoesnotexist.com/](https://thispersondoesnotexist.com/). The images provided are all AI generated and not real people. In the samples below, you can see there are some malformations that stand out.
```
Comment (Techneaux) [inline _posts/2021-07-13-randomize-slack.md:14]: found another typo. `a random avatars`

**Review by Techneaux — CHANGES_REQUESTED** [review]

**Review by Techneaux — APPROVED** [review]

**Pull request merged**